docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md

```diff
@@ -0,0 +1,493 @@
+    | **Scopes**        | Add `openid`, `profile` and `email`.                                                                                                                                                                                         |
+    | **Email**         | Use `email` as the default value                                                                                                                                                                                             |
+
+25. Leave other fields with the default values and click **Enable**. If all required values are provided, you will
+    receive a message stating that OIDC is configured successfully.
+
+    ![Example of Palette with populated input fields](/oidc-entra-id-images/enable.png)
+
+You have now successfully configured OIDC SSO in Palette. Next, validate the configuration by logging in to Palette with
+an Entra ID user account.
+
+### Validate
+
+1. Log out of Palette. To log out, click on **User Menu** in the top right corner of the screen and select **Sign Out**.
+
+2. The Palette login screen now displays a **Sign in** button and no longer presents a username and password field. If
+   this is your first time logging in with SSO, you will be redirected to the Entra ID login page. Depending on your
+   organization's SSO settings, this could be a simple login form or require MFA (Multi-Factor Authentication). Make
+   sure you log in as a user who is a member of the `palette-tenant-admins` group in Entra ID. Once authenticated, you
+   will automatically be redirected back to Palette and logged into Palette as that user.
+
+3. Navigate to the left **Main Menu** and ensure the **Tenant Settings** option is available. If you do not see the
+   **Tenant Settings** option, you are not logged in as a user who is a member of the `palette-tenant-admins` group in
+   Entra ID.
+
+   :::tip
+
+   Below the **Sign In** button, there is a link titled **SSO issues? --> Use your password**. The link can be used to
+   bypass SSO and log in with a local Palette account in case there is an issue with SSO and you need to access Palette
+   without SSO.
+
+   :::
+
+## Enable OIDC in Kubernetes Clusters With Entra ID
+
+Kubelogin is a kubectl plugin for Kubernetes OpenID Connect (OIDC) authentication. When you use kubectl, kubelogin opens
+up your browser, starts a session, and redirects you to your IDP's login site. Upon a succesfull login, you receive an
+authentication token that is used to grant you access to the cluster.
+
+In the following diagram, Entra ID is the IDP. Once Kubelogin gets a token from Entra ID or similar IDP, you can then
+use kubectl to access the Kubernetes cluster API.
+
+    ![Palette and EntraID](/oidc-entra-id-images/PalettewEntraID.png)
```

Comment:
Hi Jesse :) Would it be possible to create a diagram of our own? We may have problems using this one because of Microsoft image rights. :disappointed: